### PR TITLE
Added the kns formula.

### PR DESCRIPTION
**Pull request by koenbollen**
So people can install https://gist.github.com/koenbollen/342c138b6a77d3de6869aad68c193617

### CODE REVIEW

**Comment by koenbollen** [comment]
@koenrh @JeanMertz 👍❓

**Review by arnov — APPROVED** [review]
afaik

**Review by jewilmeer — APPROVED** [review]
lgtm 🎉

**Comment by koenbollen** [comment]
I'm going to wait for an approval by @koenrh. He should be back tomorrow.

**Review by JeanMertz — CHANGES_REQUESTED** [review]
Awesome to contribute this 👍 

I'm requesting a change, but in general, I'm extremely happy to see this take off and have more tools easily available to all developers 🎉 

As we talked about over Slack: I'd like to see the script be available somewhere under our GH organisation, so we can change it if we ever need to.

File: Formula/kns.rb
Comment (koenrh) [inline Formula/kns.rb]: I think Homebrew Core now supports installing completions:

Not tested, but I think you could add this to the `install` method, and then remove `post_install`:

```
bash_completion.install "kns.bash"
zsh_completion.install "kns.bash"
```

**Comment by koenrh** [comment]
@JeanMertz That would be nice. We could simply add a separate repository for all completions: something like https://github.com/blendle/shell-completions. Like I did here for shell-scripts: https://github.com/koenrh/shell-scripts (https://github.com/koenrh/homebrew-scripts).

**Comment by koenbollen** [comment]
@koenrh They are not completions. 😄 Just a function to call.

@JeanMertz I'll create a clone of my gist in a blendle repo soon.

File: Formula/kns.rb
Comment (koenrh) [inline Formula/kns.rb:16]: image

**Review by koenrh — APPROVED** [review]

File: Formula/kns.rb
Comment (koenbollen) [reply Formula/kns.rb:16]: I do, really. Recommend that alias!

**Review by JeanMertz — APPROVED** [review]
awesome

**Comment by JeanMertz** [comment]
@koenbollen one thing I noticed:

I installed `kubectl` using `gcloud component install kubectl`, and that's what we advice people to do in tons of `script/bootstrap` scripts as well, because that way, you get notified if an update is available.

This formula _depends_ on `kubernetes-cli`, meaning Homebrew overwrote my kubectl.

Would be nice if this was a soft-dependency, so that doesn't happen to everyone.

**Comment by koenbollen** [comment]
@JeanMertz Agreed, will fix. :+1:

**Comment by koenrh** [comment]
@koenbollen It doesn't necessarily need to be _just_ a 'completion'. See: https://github.com/ahmetb/kubectx

**Pull request merged**
